pysat/_params.py

```diff
@@ -25,10 +24,10 @@ class Parameters(object):
     ----------
     path : str
         If provided, the directory path will be used to load/store a
-        parameters file with name 'pysat_settings.json' (default=None).
+        parameters file with name 'pysat_settings.json' (default=None)
     create_new : bool
         If True, a new parameters file is created. Will be created at path
-        if provided. If not, file will be created in `pysat.pysat_dir`.
+        if provided. If not, file will be created in `~/.pysat`.
```

Comment:
```suggestion
        if provided. If not, file will be created in `.pysat` stored
        under the user's home directory.
```
be ecumenical toward windows users